hw/usb/tinyusb/syscfg.yml

```diff
@@ -33,7 +33,7 @@ syscfg.defs:
     USBD_SYSINIT_STAGE:
         description: >
             Sysinit stage for USB device functionality.
-        value: 500
+        value: 21
```

Comment:
add syscfg for this sysinit and change priority in your local target if needed

Reply:
CONSOLE_SYSINIT_STAGE Is set to 20 by default and that same level(syscfg) is used for CDC console. Tinyusb needs to be initialized before the CDC console if we want the CDC console to work fine by default, else it is going to have race conditions which is what we are seeing.

Reply:
I understand that but are you sure that changing default value for tinyusb from 500 to 21 will not introduce any side effects in other configurations (<- @kasjer)? You can add conditional restriction instead so newt will print error if CDC console is enabled and USBD_SYSINIT_STAGE >= CONSOLE_USB_CDC_SYSINIT_STAGE and then change USBD_SYSINIT_STAGE in your local config accordingly.

Reply:
I'm not sure what will happen if we move USB initialization to 21. So maybe it is safer to keep default priority as @andrzej-kaczmarek suggested.

Reply:
@andrzej-kaczmarek I am fine closing the PR if you think this is not needed. As you said I can very well fix it in my local target. I just wanted to fix it in master/upstream as well.

Reply:
Well, I have tested it and seems to be working fine. @benmccrea can second me on that.

Reply:
I think adding CONSOLE_USB_CDC_SYSINIT_STAGE still makes sense as otherwise you won't be able to set proper priority order in scenario you mentioned (correct me if I'm wrong). I'm just reluctant to change default configuration for all other cases as well if we're not sure what could be the side-effects.

Reply:
Though, if you guys feel nervous about   initializing it that early, cdc_console will have to be initialized later which will be in the 500s which will be too late. I am fine either ways.

Reply:
One more thing to consider is all hardware stuff gets initialized quite early and only tinyusb which in turn initializes the USB controller for da1469x in our specific case gets initialized in the 500s.

Reply:
Having said that I understand the concerns, I will just add CONSOLE_USB_CDC_SYSINIT_STAGE  and set it to 500+ sysinit level. I will change our targets locally to initialize both early on. We (mynewt) can probably make a well tested change later to do it early on.

Reply:
Having said that I understand the concerns, I will just add CONSOLE_USB_CDC_SYSINIT_STAGE  and set it to 500+ sysinit level. I will change our targets locally to initialize both early on. We (mynewt) can probably make a well tested change later to do it early on.

Reply:
done